.Rhistory

```diff
@@ -1,57 +1,3 @@
-ggplot(zurich) + geom_sf()
```

Comment:
remove your .Rhistory from your cached files (see #65)!

Reply:
and also, really try out the RStudio-Git-GUI, you are going to love it. I'm really surpised that it took me so long to realize this. Just overwrite your `id_rsa` I dont think much will happen.. you'll love it ❤️